bundles/org.openhab.binding.wundergroundupdatereceiver/README.md

```diff
@@ -0,0 +1,227 @@
+
+#### Synthetic channel-types. These are programmatically added:
+
+|  Channel type id       | Type                 | Channel type | Label                    | Description                                                                                                                                                | Group    |
+|------------------------|----------------------|--------------|--------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|
+| dateutc-datetime       | dateutc-datetime     | state        | Last Updated as DateTime | The date and time of the last update in UTC as submitted by the weather station converted to a DateTime value. In case of 'now', the current time is used. | Metadata | 
+| last-received-datetime | DateTime             | state        | Last Received            | The date and time of the last update.                                                                                                                      | Metadata | 
+| last-query-state       | String               | state        | The last query           | The part of the last query after the first unurlencoded '?'                                                                                                | Metadata |
+| last-query-trigger     | String               | trigger      | The last query           | The part of the last query after the first unurlencoded '?'                                                                                                | Metadata |
+
+The trigger channel's payload is the last querystring, so the following dsl rule script would send the measurements on to wunderground.com:
+
+```
+val requestQuery = receivedEvent
+sendHttpGetRequest("https://rtupdate.wunderground.com/weatherstation/updateweatherstation.php?" + requestQuery)
+```
+
+The PASSWORD, action and realtime parameters are ignored, as they are wunderground technical constants, that devices must send.
+The ID parameter is used to identify the correct thing the request pertains to, i.e. the stationId configuration value.
+
+As described by the wunderground specification a device can submit multiple values for the outdoor temperature, soil temperature, soil moisture and leaf wetness channels by inserting an index number into the name of the request parameter, fx. tempf can be temp1f, temp2f, etc.
+This is supported by the discovery mechanism, creating a channel for each of the values.
+
+# Examples
+
+### Thing file
+
+Configuration using thing and item files is not the recommended method, as you have to manually replicate the configuration, discovery produces.
+Items must be named as the request parameters in the channel type table and must have the same types.
+Items matching the request parameters submitted by your particular device must needs be discovered before being able to write appropriate item files.
```

Comment:
Items are not discovered by openHAB or do you mean that the parameter needs to be intercepted by the user first? Can you also check the grammer on this sentence?